### ISSUE TITLE
Change user language of sign in page with ui_locales option (Auth)

### ISSUE THREAD

**Issue by Forestbrook** [microsoft/appcenter#721]
Currently the sign in page can be translated to the customer's browser language by selecting 
**Enable language customization** on the **Languages** tab of the **User Flow** in Azure AB B2C.

In our apps, the user can select another language than the device default language. We would like to be able to show the sign in page in the language selected by the user.

This should be possible in MSAL by setting the **ui_locales** option as described in [Language customization in Azure Active Directory B2C](https://docs.microsoft.com/en-us/azure/active-directory-b2c/active-directory-b2c-reference-language-customization).

Could you please add an option to AppCenter Auth to set the ui_locales option?

And please mention in the documentation for this option that ‘enable language customization’ must be turned on in the User Flow!

**Comment by amchew** [comment]
Hey @Forestbrook, thanks for using Auth! 

Would you mind sharing a little more about your use case, and what are the key languages used by your apps? Also, would love to get your feedback on how the Auth experience is going for you. If you're keen to chat, please email me at amchew [at] microsoft [dot] com.

**Comment by Forestbrook** [comment]
In smaller countries (like in my case The Netherlands) a lot of people do not set the default language to their own language. For several reasons, e.g. because they don’t like English websites to be translated automatically, or just because the phone is delivered with another default language.

But when they install an app which supports their home language, like our apps do, they switch to their home language. Now when the app is in Dutch, it would be strange when the logon page is shown in English.

Azure AD B2C is prepared for that, you can provide the query string parameter, ui_locales, with the language you want the logon page to show.

What I need in Auth is an option to set that ui_locales query string parameter.

**Comment by amchew** [comment]
Hey @forestbrook, thanks for the elaboration! Will speak to our team internally to explore providing the parameter, and will update again.

**Comment by Zakeelm** [comment]
@Forestbrook  Thank you so much for your feedback. We announced this week in a [blog post]( https://devblogs.microsoft.com/appcenter/app-center-mbaas-retirement/) that we’re retiring the Auth and Data services later this year and recommending customers transition to Azure Active Directory B2C and Azure Cosmos DB instead. Let me know if you have any additional questions about this transition and i'll make sure to get them answered! Closing this issue.
